MangaManager/src/MetadataManager/GUI/SettingsWidgetManager.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
```

Comment:
I had planned to add settings to the TLI.

I'm wondering whether we should tweak this to support CLI it or just make a new implementation for it.

Reply:
CLI can inherit the settings from extensions without any need to talk to SettingsWidgetManager btw.